gold-phone-input.html

```diff
@@ -157,9 +157,27 @@
       }
     },
 
-    observers: [
-      '_computeValue(value)'
-    ],
+    listeners: {
+      'input': '_onInput'
+    },
+
+    ready: function() {
+      this.addEventListener('blur', function() {
+        this._handleAutoValidate();
```

Comment:
Hrm, after trying it out, I guess the context is implicitly bound to the element? I didn't know this was the case.